manifest-lock.overrides.ppc64le.yaml

```
@@ -0,0 +1,23 @@
+# This lockfile should be used to pin to a package version (`type: pin`) or to
+# fast-track packages ahead of Bodhi (`type: fast-track`). Fast-tracked
+# packages will automatically be removed once they are in the stable repos.
+#
+# IMPORTANT: YAML comments *will not* be preserved. All `pin` overrides *must*
+# include a URL in the `metadata.reason` key. Overrides of type `fast-track`
+# *should* include a Bodhi update URL in the `metadata.bodhi` key and a URL
+# in the `metadata.reason` key, though it's acceptable to omit a `reason`
+# for FCOS-specific packages (ignition, afterburn, etc.).
+
+packages:
+  grub2-ppc64le:
+    evra: 1:2.12-10.fc41.ppc64le
+    metadata:
+      bodhi: https://bodhi.fedoraproject.org/updates/FEDORA-2024-7d58433dd5
+      reason: https://github.com/coreos/fedora-coreos-tracker/issues/1802
+      type: fast-track
+  grub2-ppc64le-modules:
+    evra: 1:2.12-10.fc41.ppc64le
```

I think this one is `noarch` for some reason:

```suggestion
    evra: 1:2.12-10.fc41.noarch
```

```
$ grep -A 1 grub2- manifest-lock.ppc64le.json 
    "grub2-common": {
      "evra": "1:2.12-9.fc41.noarch",
--
    "grub2-ppc64le": {
      "evra": "1:2.12-9.fc41.ppc64le",
--
    "grub2-ppc64le-modules": {
      "evra": "1:2.12-9.fc41.noarch",
--
    "grub2-tools": {
      "evra": "1:2.12-9.fc41.ppc64le",
--
    "grub2-tools-minimal": {
      "evra": "1:2.12-9.fc41.ppc64le",
```